go.mod

```diff
@@ -13,7 +13,7 @@ require (
 	github.com/cespare/cp v0.1.0
 	github.com/cloudflare/cloudflare-go v0.14.0
 	github.com/consensys/gnark-crypto v0.4.1-0.20210426202927-39ac3d4b3f1f
-	github.com/crate-crypto/go-ipa v0.0.0-20230601170251-1830d0757c80
+	github.com/crate-crypto/go-ipa v0.0.0-20230626131944-6a9b06cf26df
```

Comment:
@gballet, should we target https://github.com/gballet/go-verkle/pull/354 this now, right?

Reply:
yes